### PR TITLE
#7240 Extract one line which underscores name of template file into a seperate method

### PR DESCRIPTION
**Pull request by ghost**
Extracted the line which underscores the name of template file into a
seperate line so that people can override it.

### CODE REVIEW

**Comment by NaranjaWeb** [comment]
I could not inherit the preceding pull request and made another one.

File: src/View/View.php [hunk 1]
```diff
@@ -1054,6 +1056,16 @@ protected function _getViewFileName($name = null)
     }
 
     /**
+     * Change the name of a view template file into underscored format.
+     *
+     * @param string $name Name of file which should be inflected.
+     * @return string File name after conversion
+     */
+    protected function _inflectViewFileName($name) {
```
Comment (HavokInspiration) [inline src/View/View.php]: Opening ``{`` should be on a new line

**Comment by NaranjaWeb** [comment]
Thank you, and I feel sorry for easy mistakes.

**Comment by markstory** [comment]
Looks reasonable to me.

**Comment by lorenzo** [comment]
Thanks @NaranjaWeb

**Pull request merged**